src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerStepExecution.java

```diff
@@ -107,7 +107,7 @@ public boolean start() throws Exception {
     @Override
     public void stop(@Nonnull Throwable cause) throws Exception {
         LOGGER.log(Level.FINE, "Stopping container step.");
-        closeQuietly(getContext(), client, decorator);
+        closeQuietly(getContext(), decorator);
```

Comment:
BTW I would recommend getting rid of `Resources.closeQuietly` and switch to Java 7 suppressed exceptions—unless you have a specific reason to _expect_ an exception from some `close` method which you have a specific reason to _want to recover from_, in which case write that explicitly. But the default should be to propagate failures.

Also I recommend all `StepExecution` subtypes delegate to `super.stop` unless you really know what you are doing.